index.d.ts

```diff
@@ -41,354 +41,356 @@ type Pillar = Theme;
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type DesignTypesObj = { [key in Design]: any };
 
+type Palette = { [key: string]: any };
```

Comment:
Do you not know the type of these fields?

Reply:
We actively decided to not type this object to reduce the time needed to expand it's usage but, for sure, at some point the advantages of autocomplete will outweigh the cost of updating two things to add a new property